code/datums/supply_packs/_supply_packs.dm

```diff
@@ -34,14 +35,14 @@ var/list/all_supply_groups = list(
 	var/name = "Basic supply pack."
 	var/list/contains = list()
 	var/manifest = ""
-	var/cost = RO_PRICE_NORMAL
+	var/cost = 7 				/// Determine the price of the crate. Give it a value between 7 and 100 to avoid breaking the game.
```

Comment:
Should be 40 here then!

Reply:
i put a seven because it's the lowest number we should imput for cost.